builds/jazz-build-module/sls-app/sbr.groovy

```diff
@@ -715,6 +715,7 @@ class SBR_Whitelist_Constraint implements SBR_Constraint {
       case "events": return whitelistValidator.validateWhitelistEvents(val); break;
       case "plugins": return whitelistValidator.validateWhitelistPlugins(val); break;
       case "actions": return whitelistValidator.validateWhitelistActions(val); break;
+      case "iamManagedPolicies": return whitelistValidator.validateWhitelistIamManagedPolicies(val); break;
```

Comment:
We need to change it as a separate effort to surface the issue during validation instead of just returning true or false.